test/test_PokerSetup_2.py

```diff
@@ -0,0 +1,39 @@
+from testing_base import *
```

Comment:
Finding test cases for pregame setup is kinda tricky but maybe add some assertions for the expected result of the setup.